### DISCUSSION TITLE
Add additional scenery types in Track Designer

### DISCUSSION THREAD

**Discussion by skylerwlewis** [OpenRCT2/OpenRCT2#18529] [Feature requests] [answered]
### Operating System

macOS 10.14.6, (x86-64)

### OpenRCT2 build

OpenRCT2, v0.4.2 (8ceea45 on HEAD)

### Base game

RollerCoaster Tycoon Classic

### Area(s) with this issue?

Track Designer

### Describe the issue

There seems to be no ability to select additional scenery elements (Roman, Greek, Egyptian, etc.) so they are available for use in Track Designer in OpenRCT2. In RCTC, additional scenery elements are able to be selected in the Object Selection window, I was curious could we have something similar in OpenRCT2? It looks like OpenRCT2 has added paths and scenery into Track Designer, it would be much more useful for tweaking scenery if the ability to select additional scenery themes was added.

It looks like the Scenario Editor Object Selection Window in OpenRCT2 currently allows users to select scenery, paths & benches/lights. I wonder if same window layout could be used for the Park Designer, only without the Park Entrance tab and Water tab, since I believe every other selection in the Scenario Editor Object Selection Window would be able to be saved in a .td6 file.

Thanks!

### Steps to reproduce

1. Add the attached track file to your custom tracks folder.
2. Launch OpenRCT2 and open Track Designer.
3. In the Object Selection window, Select "Spinning Wild Mouse" to make it available.
4. Click Forward to Next Step: Track Designer.
5. Select Spinning Wild Mouse.

You should see a "Sugar Mice" track layout, but it contains scenery that is unavailable. I created this track file using OpenRCT2, but it is not placable in Track Designer because its scenery is not able to be used.

### Attachments

[Sugar Mice.td6.zip](https://github.com/OpenRCT2/OpenRCT2/files/9946230/Sugar.Mice.td6.zip)

**Comment by skylerwlewis** [comment]
Sort of related to #6844 & #13587

**Comment by duncanspumpkin** [comment]
This is a feature request not a bug.

**Comment by skylerwlewis** [comment, marked as answer]
Implemented by https://github.com/OpenRCT2/OpenRCT2/pull/13587.
